pkg/cmd/cli/describe/printer.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED"}
+	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED", "REASON"}
```

Comment:
@rhcarvalho so I would rather prefer to have one less column and add the reason string as part of the status... Meaning if the status will be != Complete and reason is not empty, show it as:

"Failed (Reason)"